### PR TITLE
[GEN-628] Update PFS status variables

### PR DESCRIPTION
**Pull request by thomasyu888**
* Make sure PFS status variables are remapped to the correct values
* Use convenience functions for the remapping
* Only BLADDER supports TIMELINE-PERFORMANCE right now

### CODE REVIEW

File: geniesp/bpc_config.py [hunk 1]
```diff
@@ -3,39 +3,43 @@
...
     _SPONSORED_PROJECT = "PANC"
+    _exclude_files = ["data_timeline_performance_status.txt"]
 
 
 class Prostate(BpcProjectRunner):
     """Prostate BPC sponsored project"""
 
     # Sponsored project name
     _SPONSORED_PROJECT = "Prostate"
+    _exclude_files = ["data_timeline_perforamnce_status.txt"]
```
Comment (Chelsea-Na) [inline geniesp/bpc_config.py]: Should be: ["data_timeline_performance_status.txt"]

File: geniesp/bpc_redcap_export_mapping.py [hunk 2]
```diff
@@ -471,13 +471,36 @@ def hack_remap_laterality(df_patient_subset: pd.DataFrame) -> pd.DataFrame:
...
+                    if col.startswith('OS') and col.endswith("STATUS")}
+    return df.replace(remap_values)
+
+
+def remap_pfs_values(df: pd.DataFrame):
+    """Remap PFS numerical values to string values
+    0 -> 0:CENSORED
+    1 -> 1:PROGRESSION
+    """
+    remap_values = {col: {0: "0:CENSORED", 1: "1:PROGRESSED"}
```
Comment (Chelsea-Na) [inline geniesp/bpc_redcap_export_mapping.py:490]: Should be "1:PROGRESSION" ?

**Review by Chelsea-Na — COMMENTED** [review]
A couple of minor spelling mistakes may be present.

File: geniesp/bpc_config.py (hunk 1 above)
Comment (thomasyu888) [inline geniesp/bpc_config.py]:
```suggestion
    _exclude_files = ["data_timeline_performance_status.txt"]
```

File: geniesp/bpc_redcap_export_mapping.py [hunk 3]
```diff
@@ -471,13 +471,36 @@ def hack_remap_laterality(df_patient_subset: pd.DataFrame) -> pd.DataFrame:
...
+    remap_values = {col: {0: "0:LIVING", 1: "1:DECEASED"}
+                    for col in df.columns
+                    if col.startswith('OS') and col.endswith("STATUS")}
+    return df.replace(remap_values)
+
+
+def remap_pfs_values(df: pd.DataFrame):
+    """Remap PFS numerical values to string values
+    0 -> 0:CENSORED
+    1 -> 1:PROGRESSION
```
Comment (thomasyu888) [inline geniesp/bpc_redcap_export_mapping.py]:
```suggestion
    1 -> 1:PROGRESSED
```

**Review by Chelsea-Na — APPROVED** [review]

**Pull request merged**